main.c

```diff
@@ -306,7 +315,7 @@ sexp run_main (int argc, char **argv) {
   sexp_uint_t heap_size=0, heap_max_size=SEXP_MAXIMUM_HEAP_SIZE;
   sexp out=SEXP_FALSE, ctx=NULL, ls;
   sexp_gc_var4(tmp, sym, args, env);
-  args = SEXP_NULL;
+  args = NULL;
```

Comment:
Why are you changing this?  You set it to SEXP_NULL below anyway.

Reply:
Must be my mistake.